r3bdata/neulandData/R3BNeulandPoint.h

```diff
@@ -37,32 +39,44 @@ class R3BNeulandPoint : public FairMCPoint
                     const Double_t length,
                     const Double_t eLoss,
                     const UInt_t EventId,
-                    const Double_t lightYield)
-        : FairMCPoint(trackID, detID, pos, mom, tof, length, eLoss, EventId)
-        , fLightYield(lightYield)
+                    const Double_t lightYield,
+                    const int particle_id,
+                    const int parent_particle_id)
```

Comment:
We could just tell static analyser to ignore this warning by adding `NOLINTBEGIN` and `NOLINTEND`
```c++
// NOLINTBEGIN
R3BNeulandPoint(const Int_t trackID,
                    const Int_t detID,
                    const TVector3& pos,
                    const TVector3& mom,
                    const Double_t tof,
                    const Double_t length,
                    const Double_t eLoss,
                    const UInt_t EventId,
                    const Double_t lightYield,
                    const int particle_id,
                    const int parent_particle_id)
// NOLINTEND
```